src/OnboardingSPA/components/Header/index.js

```diff
@@ -1,27 +1,31 @@
 import { Slot } from '@wordpress/components';
 import { Fragment, Suspense } from '@wordpress/element';
 import { useSelect } from '@wordpress/data';
+import classNames from 'classnames';
 
 import { store as nfdOnboardingStore } from '../../store';
 import {
 	HEADER_CENTER,
 	HEADER_END,
 	HEADER_START,
 	HEADER_TOP,
+	HEADER_SITEGEN,
 } from '../../../constants';
 
 const Header = () => {
-	const { headers, headerActiveView, isHeaderEnabled } = useSelect(
-		( select ) => {
+	const { headers, headerActiveView, currentStep, isHeaderEnabled } =
+		useSelect( ( select ) => {
 			return {
+				currentStep: select( nfdOnboardingStore ).getCurrentStep(),
 				headers: select( nfdOnboardingStore ).getHeaders(),
 				headerActiveView:
 					select( nfdOnboardingStore ).getHeaderActiveView(),
 				isHeaderEnabled: select( nfdOnboardingStore ).isHeaderEnabled(),
 			};
-		}
-	);
-
+		} );
+	const isSiteGenEditor =
```

Comment:
@officiallygod can you review is this is a right way or can this be handled differenlty? I am adding classname just for the editor page so that I can add a background color to the header on the editor page.